### ISSUE TITLE
Cut to black instead of fading to black at the beginning of a new day

### ISSUE THREAD

**Issue by Pib0** [leoetlino/project-restoration#67]
This isn't really a feature request so much as it is a design choice. Majora's Mask 3D changed the new day transition so it fades to black before the new day splash screen. That's not necessarily a bad thing, I just personally think it's not as cool. Any chance it can be changed back to the way it was originally?

**Comment by GeneraLight** [comment]
I agree with the instant new day screen transition and Goron camera angles. The cut to black like in the original N64 versions should theoretically be a tiny bit faster too. The camera angle you specified would also make Goron rolling appear faster.

**Comment by nr1995** [comment]
I agree the cut to black was much more powerful thematically.
Maybe they did it so people wouldn't think their 3DS just crashed  XD

**Comment by mic88may-sys** [comment]
You can get the camera looking further away by using a 3D depth altering cheat code from gbatemp @Pib0
